src/annotator/test/host-test.js

```diff
@@ -0,0 +1,122 @@
+import Host from '../host';
+
+describe('Host', () => {
+  const sandbox = sinon.createSandbox();
+  const hostConfig = { pluginClasses: {} };
+
+  let CrossFrame;
+  let fakeCrossFrame;
+
+  const createHost = (config = {}, element = null) => {
+    config = Object.assign(
+      { sidebarAppUrl: '/base/annotator/test/empty.html' },
+      hostConfig,
+      config
+    );
+    if (!element) {
+      element = document.createElement('div');
+    }
+    return new Host(element, config);
+  };
+
+  beforeEach(() => {
+    // Disable any Host logging.
+    sandbox.stub(console, 'log');
```

Comment:
Do you have a preference between sandbox or

`sinon.stub(console, 'log');`
`console.log.restore()` 

Just wondering when to use one or the other

Reply:
If you're only stubbing one or two functions, `sinon.stub(...)` is simpler. If you're stubbing a lot of methods, a sandbox provides a convenient way to undo all the stubs at once IIRC. If you find yourself stubbing a lot of methods though, you might want to consider an alternative approach such as injecting a fake version of the object that has the stubbed methods (eg. like we do with `window` in a few places)